### PR TITLE
First pass at building agent windows installer on GHA

### PR DESCRIPTION
**Pull request by dlobue**
DSET-419

### CODE REVIEW

**Comment by Kami** [comment]
Just a heads up - IIRC, this will also require updating Jenkins release jobs (IIRC, we download built artifacts from the Jenkins release job).

But @ArthurKamalov should be able to confirm that for sure.

File: .github/workflows/agent-build-windows.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@12aca0a884f6137d619d6a8a09fcc3406ced5281 # v4.0.0
+        with:
+          cancel_others: 'true'
+          github_token: ${{ github.token }}
+
+  build-windows-package:
+    name: Build windows MSI installer
+    runs-on: windows-latest
```
Comment (Kami) [inline .github/workflows/agent-build-windows.yml:29]: Let's please add skip if check here.

Comment (dlobue) [reply .github/workflows/agent-build-windows.yml:31]: Done

**Comment by ArthurKamalov** [comment]
> Just a heads up - IIRC, this will also require updating Jenkins release jobs (IIRC, we download built artifacts from the Jenkins release job).
> 
> But @ArthurKamalov should be able to confirm that for sure.

Yes, I already made those changes for new linux packages to be downloaded from GHA, and will do the same for Windows Installer.

**Comment by dlobue** [comment]
For the record: I just verified that the installer works with no issues in a windows VM. Took me forever to get a working windows VM. In the end, I had to use a VM I already had on my personal laptop.

**Review by ArthurKamalov — APPROVED** [review]
LGTM :+1:

**Pull request merged**